pkg/services/ngalert/state/manager.go

```diff
@@ -175,6 +175,14 @@ func (st *Manager) setNextState(alertRule *ngModels.AlertRule, result eval.Resul
 	case eval.Pending: // we do not emit results with this state
 	}
 
+	// Set Resolved property so the scheduler knows to send a postable alert
+	// to alert manager.
+	if oldState == eval.Alerting && currentState.State == eval.Normal {
```

Comment:
can just be `currentState.Resolved = oldState == eval.Alerting && currentState.State == eval.Normal` I think